### ISSUE TITLE
FABC x_data argument

### ISSUE THREAD

**Issue by hughhop** [barahona-research-group/RamanSPy#21]
in the FABC baseline correction object there is an x_data input argument. however this conflicts with the x_data argument supplied to np.apply_along_axis baseline.py L33 and this causes `TypeError numpy.apply_along_axis() got mulitple values for the keyword argument 'x_data'`

python 3.11
ramanspy 0.2.10

how to recreate the issue:
```
import ramanspy as rp
import numpy as np
s = rp.Spectrum(np.random.rand(1000), np.linspace(300, 2000, 1000))
s2 = rp.preprocessing.baseline.FABC().apply(s)
```
output 

`TypeError numpy.apply_along_axis() got mulitple values for the keyword argument 'x_data'`

**Comment by hughhop** [comment]
none of the other baseline classes include x_data in the class definition since it is in the base class

Ive raised a PR where this input argument from FABC is removed.

#20

**Comment by dgeorgiev21** [comment]
The PR has been merged into the v0.2.11 branch. See commit [ed8ab18](https://github.com/barahona-research-group/RamanSPy/commit/ed8ab188f1aa416d018fe1097b70071bd32b0965). 

Thank you for the assistance!
